_minutes/2023-01-05-wecg.md

```diff
@@ -0,0 +1,133 @@
+ * [simeon] Chromium would prefer to keep the Action API as it is. I think it would be best to discuss this request as a new API.
+ * [giorgio] Could we repurpose the windows.create() API (type: “panel”) to support this use case?
+ * [rob] That resulting API would not be heavy-weight. On Firefox for Android the windows API doesn't even exist.
+ * [timothy] On iOS windows can be accessed but not closed.
+ * [simeon] Random thought: maybe a "browser.modal" extension API to open a modal/lightbox etc. Allows browsers to choose appropriate presentation of this content for the device/platform.
+ * [timothy] I'm supportive of that. Reminds me of the `<dialog>` element which is well-supported.
+ * [rob] I'm supportive of the capability, but there are a lot of different directions we could go. I think we need an API proposal in order to discuss something specific. The API in its current form has an unsurmountable issue (inability to associate the UI with the extension button), and we would therefore vote against it. There is a favorable sentiment for the feature in a different shape.
+
+[Issue 162](https://github.com/w3c/webextensions/issues/162): DNR proposal: disable individual static rules. Increase limit to 20k rules ([comment](https://github.com/w3c/webextensions/issues/162#issuecomment-1318618540)).
+
+ * [simeon] Is Felipe here? No. Suggest we table this until next session. Before we move on, any concerns on the shape of the APi?
+ * [rob] Shape of the API looks good, as proposed at: https://github.com/w3c/webextensions/issues/162#issuecomment-1245662951
+ * [timothy] Shape of the API looks good. I can see a feasible path towards an implementation of this in Safari.
+ * [rob] Timothy, please label this as “supportive: safari”? I'll put the “supportive: firefox” label on it.
+
+[Issue 336](https://github.com/w3c/webextensions/issues/336): Inconsistency: action.setIcon() & action.setBadgeBackgroundColor()
+
+ * [simeon] These issues are minor and very specific, let's continue the discussion async in the issue.
+
+[Issue 337](https://github.com/w3c/webextensions/issues/337): Inconsistency: CSP for inline web page elements added by extension
+
+ * [rob] The issue here is the lack of clarity and agreement between browsers on what the CSP of content scripts cover. I recently expanded [the documentation on MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_Security_Policy#csp_for_content_scripts), which is basically that the CSP in content scripts is browser-specific. I know that Chrome has made an effort to cover as much by the content script CSP as possible, and that Firefox mostly follows the web page's CSP with some exceptions. How about Safari?
+ * [timothy] The CSP mostly follows the content script of the page.
+ * [simeon] CSP for content scripts exists to block remotely hosted code.
+ * [timothy] Haven't thought of that.
```

Comment:
We should remove this line. I think I was responding to something else @dotproto was saying that wasn't captured in the minutes.